tests/integration_tests/cli_query_toml/run.sh

```
@@ -0,0 +1,154 @@
+	echo "PASS: Test 16 - Short flag -o toml works"
+
+	# Cleanup changefeeds
+	cdc_cli_changefeed --changefeed-id "cf-default" remove
+	cdc_cli_changefeed --changefeed-id "cf-overrides" remove
+	cdc_cli_changefeed --changefeed-id "cf-large-filter" remove
+
+	cleanup_process $CDC_BINARY
+}
+
+trap 'stop_test $WORK_DIR' EXIT
+run $*
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)